R/pkg/tests/fulltests/test_mllib_classification.R

```diff
@@ -313,7 +313,7 @@ test_that("spark.mlp", {
   # Test predict method
   mlpTestDF <- df
   mlpPredictions <- collect(select(predict(model, mlpTestDF), "prediction"))
-  expect_equal(head(mlpPredictions$prediction, 6), c("1.0", "0.0", "0.0", "0.0", "0.0", "0.0"))
+  expect_equal(head(mlpPredictions$prediction, 6), c("0.0", "1.0", "1.0", "1.0", "1.0", "1.0"))
```

Comment:
This is due to the change of how we sort string labels with same frequency under the setting of frequencyDesc/Asc.

Reply:
I checked the predictions. All `0.0` -> `1.0`, and all`1.0` -> `0.0`.